lib/BridgeAbstract.php

```diff
@@ -238,6 +245,52 @@ function($i){ return $i['name']; }, // Just display parameter names
 
 	}
 
+	/**
+	 * Loads configuration for the bridge
+	 *
+	 * Returns errors and aborts execution if the provided configuration is
+	 * invalid.
+	 *
+	 * @return void
+	 */
+	public function loadBridgeConfiguration() {
+		foreach(static::CONFIGURATION as $optionName => $optionValue) {
+
+			$configurationOption = Configuration::getConfig(get_class($this), $optionName);
+
+			if($configurationOption !== null) {
+				$this->configuration[$optionName] = $configurationOption;
+				continue;
+			}
+
+			if($optionValue['required']) {
```

Comment:
This will give an error if `required` is not defined.

```suggestion
			if(isset($optionValue['required']) && $optionValue['required'] === true) {
```